changelog/NOTE.md

```diff
@@ -7,100 +7,100 @@ reva users. The changes are ordered by importance.
 Summary
 -------
 
- * Fix #3109: Fix missing check in MustCheckNodePermissions
- * Fix #3086: Fix crash in ldap authprovider
- * Fix #3094: Allow removing password from public links
- * Fix #3096: Fix user filter
- * Fix #3091: Project spaces need no real owner
- * Fix #3088: Use correct sublogger
- * Enh #3123: Allow stating links that have no permissions
- * Enh #3087: Allow to set LDAP substring filter type
- * Enh #3098: App provider http endpoint uses Form instead of Query
- * Enh #3133: Admins can set quota on all spaces
- * Enh #3117: Update go-ldap to v3.4.4
- * Enh #3095: Upload expiration and cleanup
+*   Fix #3109: Fix missing check in MustCheckNodePermissions
```

Comment:
This file is autogenerated

Reply:
Then template needs to be adjusted. Any idea where I can find it?